README.md

```diff
@@ -1 +1,39 @@
-# Portfolio-Mobile-first-project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project 1 Portfolio 
+
+> Web design for mobile version
+
```

Comment:
- [ ] Kindly make sure you have a detailed description telling what the project is about. as it will help your work to be documented professionally.